automatic/poi/README.md

```diff
@@ -0,0 +1,5 @@
+﻿# <img src="https://cdn.rawgit.com/poooi/poi/master/assets/icons/poi.png" width="48" height="48"/> [poi](https://chocolatey.org/packages/poi)
+
+[POI](https://poi.io) is a scalable KanColle browser and tool.
+
+More information could be obtained [here](https://github.com/poooi/poi/blob/master/README.md)
```

Comment:
This description is way too little, please add more description to the Readme.
As well as adding a `## Features` section of the different features of the software.

Reply:
Fixed in 845b10a